src/mediapipe_internal/mediapipegraphexecutor.hpp

```diff
@@ -49,6 +49,7 @@ class MediapipeGraphExecutor {
     ::mediapipe::Timestamp currentStreamTimestamp{0};
 
     Status partialDeserialize(std::shared_ptr<const ::inference::ModelInferRequest> request, ::mediapipe::CalculatorGraph& graph);
+    Status partialSerialize(std::string name, ::inference::ModelInferResponse& response, const ::mediapipe::Packet& packet) const;
```

Comment:
Do not copy the name
```suggestion
    Status partialSerialize(const std::string& name, ::inference::ModelInferResponse& response, const ::mediapipe::Packet& packet) const;
```